### PR TITLE
🔧 Add changelog URL to `pyproject.toml`, shows in PyPI

### PR DESCRIPTION
**Pull request by Pierre-VF**
The URL of the changelog was added to the project URLs in order to make it easier to see latest changes when coming from PyPi. The english version was used as a target, given that all other links are to english materials.

### CODE REVIEW

**Review by Kludex — APPROVED** [review]

**Comment by Pierre-VF** [comment]
> That's a good idea. Will it show up here?
> image
> 
> 

Yes, that's exactly the point 🙂

**Review by alejsdev — APPROVED** [review]

**Comment by alejsdev** [comment]
I updated the URL to go to the Changelog in the official docs :smile: :rocket:

File: pyproject.toml [hunk 1]
```diff
@@ -49,6 +49,7 @@ dependencies = [
 [project.urls]
 Homepage = "https://github.com/fastapi/fastapi"
 Documentation = "https://fastapi.tiangolo.com/"
+Changelog = "https://fastapi.tiangolo.com/release-notes/"
 Repository = "https://github.com/fastapi/fastapi"
```
Comment (frost-nzcr4) [inline pyproject.toml:53]: I also suggest adding an `Issues = "https://github.com/fastapi/fastapi/issues"` to not return to this a second time.

**Review by frost-nzcr4 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (Pierre-VF) [reply pyproject.toml:54]: Good idea ! I just added the suggested link

**Review by frost-nzcr4 — APPROVED** [review]

**Comment by tiangolo** [comment]
Nice, thank you! :rocket:

**Pull request merged**